python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -26,6 +28,46 @@
 from feathub.registries.registry import Registry
 from feathub.table.table_descriptor import TableDescriptor
 
+ENABLE_EMPTY_WINDOW_OUTPUT_CONFIG = (
+    "sdk.sliding_feature_view.enable_empty_window_output"
+)
+ENABLE_EMPTY_WINDOW_OUTPUT_DOC = (
+    "If it is True, when the sliding window becomes emtpy, it outputs zero value for "
+    "aggregation function SUM and COUNT, and output None for other aggregation "
+    "function. If it is False, the sliding window doesn't output anything when the "
+    "sliding window becomes empty."
+)
+
+ENABLE_STATIC_WINDOW_OUTPUT_CONFIG = (
+    "sdk.sliding_feature_view.enable_static_window_output"
```

Comment:
Could we change this config name to `sdk.sliding_feature_view.skip_same_window_output`?